exercism_test_helper/lib/meta/test_parser/test_suite.ex

```diff
@@ -1,4 +1,4 @@
 defmodule Meta.TestParser.TestSuite do
   @derive {Jason.Encoder, only: [:tests]}
```

Comment:
I think you need to make this line: 
```suggestion
  @derive {Jason.Encoder, only: [:version, :tests]}
```
so that it will be in the outputted json file